jobs/env/ci-cri-containerd-e2e-gce.env

```diff
@@ -0,0 +1,20 @@
+### job-env
+# This list should match the list in kubernetes-pull-build-test-e2e-gce.
+NUM_NODES=4
+# For now explicitly test etcd v2 mode in this suite.
+STORAGE_BACKEND=etcd2
+TEST_ETCD_IMAGE=2.2.1
+TEST_ETCD_VERSION=2.2.1
```

Comment:
(do you still wanna test against old etcd?, up to you)

Reply:
I see that all tests are still using this.

Reply:
yeah the e2e-gce uses this which I'm not sure why, anyway

Reply:
At least `ci-kubernetes-e2e-gci-gce.env` is still using it.